.github/workflows/e2e_tests.yml

```diff
@@ -325,7 +325,7 @@ jobs:
         uses: cypress-io/github-action@v6.10.2
         with:
           install: false
-          browser: chrome
+          browser: chromium
```

Comment:
```
      uses: browser-actions/setup-chrome@v2
        with:
          chrome-version: 140
      uses: cypress-io/github-action@v6.10.2
        with:
          install: false
          browser: chrome-for-testing
```

Reply:
That's what I've tested just before https://github.com/3liz/lizmap-web-client/actions/runs/19568152240/job/56035070077?pr=6244#step:29:50

But I think that the `chrome-for-testing` parameter has been introduced after our Cypress version.